### PR TITLE
Bump ot version to 0.28.0

### PR DESCRIPTION
**Pull request by pmalek-sumo**

### CODE REVIEW

**Comment by pmalek-sumo** [comment]
I've created #108 to track changes required in `pkg/processor/cascadingfilterprocessor/processor_test.go` in order to reenable the tests there

File: pkg/processor/cascadingfilterprocessor/processor_test.go [hunk 1]
```diff
@@ -401,16 +405,19 @@ func TestMultipleBatchesAreCombinedIntoOne(t *testing.T) {
 
 	expectedSpanIds := make(map[int][]pdata.SpanID)
 	expectedSpanIds[0] = []pdata.SpanID{
-		tracetranslator.UInt64ToSpanID(uint64(1)),
+		// TODO
+		// tracetranslator.UInt64ToSpanID(uint64(1)),
```
Comment (pmm-sumo) [inline pkg/processor/cascadingfilterprocessor/processor_test.go:409]: Not feeling very comfortable with keeping this commented, there should be a simple way to translate those even if the function got removed

Comment (pmalek-sumo) [reply pkg/processor/cascadingfilterprocessor/processor_test.go]: Done, copied over code from idutils related to big endian conversion

**Review by sumo-drosiek — APPROVED** [review]
LGTM

**Review by pmm-sumo — APPROVED** [review]

**Pull request merged**